### DISCUSSION TITLE
New UI too large. How to change back?

### DISCUSSION THREAD

**Discussion by Sophodot** [smplayer-dev/smplayer#177] [Q&A] [answered]
Windows 10, 64-bit, Portable Edition
On the left there's version 21.8.0 and on the right version 21.1.0. As you can see, the newer version uses a larger font size, which while I am sure people with poor eyesight will appreciate, I simply don't need. I also really don't like the larger time slider and status bar. They now take up too much space at the bottom of the window. Side-by-side there's also a noticeable color change which I don't mind.

Please, is there any setting I can change to restore the previous size of the UI?
![Screenshot](https://user-images.githubusercontent.com/89133350/129883673-9d5f6f45-6747-4872-b1e6-fe496363a4af.jpg)

**Comment by smplayer-dev** [comment, marked as answer]
I didn't notice the size was different but in preferences -> interface -> High DPI you can adjust the scale.

**Reply by Sophodot** [reply]
I had tried the DPI settings but what I think is happening is that in the new version that setting is able to scale every element while in the previous one some elements were omitted. 0.8 with icon size increased to compensate (although this icon set isn't any larger) seems to bring me closest to what I previously had, though the bars at the bottom are still larger than what they used to be.
![image](https://user-images.githubusercontent.com/89133350/129902964-4c726f8c-bb9a-4620-973a-4eb76dd241a4.png)

However, the text rendering seems broken now, like it has too sharp edges, compared to when the scale factor is on auto. Either way, this scaling mismatch is all caused by me using 150% zoom in Windows. The new version of SMPlayer handles levels above 100% differently.
![image](https://user-images.githubusercontent.com/89133350/129905142-f85e6e88-48c7-4563-be46-e3d5e1c92960.png)

**Reply by Sophodot** [reply]
I come back with another post because I've found a possibly related problem.
On the left I have a 4K screen at 150% zoom. SMPlayer looks as it should on it.
![Screenshot (9)](https://user-images.githubusercontent.com/89133350/131179684-b8903119-7ebb-49ff-a6fd-4ca156b7ed00.png)
Moving SMPlayer to the second screen, a 1080p at 100% zoom the entire GUI turns into this. Some of the options can be clicked, but the text is mostly broken and invisible.
![Screenshot (10)](https://user-images.githubusercontent.com/89133350/131179710-8008012f-b60d-4461-818b-5c5b83747efa.png)
This only happens with a user-set scale factor. Setting it to Auto maintains the integrity of the GUI on either screen. Disabling support for high DPI screens also fixes it.

Also with Auto scale factor, when dragging the window by the video area and crossing the between the two screens it snaps to a different position, and the dragging cursor can end up outside the window. Sometimes the entire player would snap outside the desktop borders with no obvious way to bring it back into view. Dragging the player window by the title bar will always keep it in place when crossing between screens.

**Comment by pizzadude** [comment]
Try deleting the config files for smplayer, that fixed it for me on Linux.

**Reply by Sophodot** [reply]
Tried it, doesn't seem to change anything for me.

**Comment by tomasz1986** [comment]
I've got the exact same problem here. The first monitor is set to 150% DPI, and the second one to 125%. The only way to make SMPlayer look properly on both screens seems to be to actually **uncheck** the `Enable support for high DPI screens`. Then, after restarting, the program fonts and icons have proper size on either of the screens.
